src/emulator/gui/src/ui_main_menubar.cpp

```diff
@@ -38,6 +38,21 @@ void DrawMainMenuBar(HostState &host) {
             ImGui::EndMenu();
         }
 
+        if (ImGui::BeginMenu("Configuration")) {
+            ImGui::PushStyleColor(ImGuiCol_Text, GUI_COLOR_TEXT_MENUBAR_OPTIONS);
+            if (ImGui::BeginMenu("Emulation Mode")) {
```

Comment:
Sorry, forgot to leave this comment:
"Emulation Mode" sounds a bit too generic, maybe something like "Emulated Platform"?

Reply:
Yeah, "Emulated Console" sounds better.